FogFog-iOS/FogFog-iOS/Presentation/Map/MapViewController.swift

```diff
@@ -7,23 +7,126 @@
 
 import UIKit
 
-class MapViewController: UIViewController {
+import SnapKit
+import Then
+import GoogleMaps
+import CoreLocation
 
+final class MapViewController: BaseViewController {
+    
+    private lazy var navigationView = NavigationView()
+    private let camera = GMSCameraPosition(latitude: 37.54330366639085, longitude: 127.04455548501139, zoom: 12)
+    private var mapView = GMSMapView()
+    private var myMarker = GMSMarker()
+    private var locationManager: CLLocationManager = CLLocationManager()
```

Comment:
```suggestion
    private var locationManager = CLLocationManager()
```

p2;
타입 명시가 필요한 상황이 아니라면 다른 객체 선언한 것처럼 스타일을 맞춰주는 것도 좋을 것 같아요!

Reply:
넵 ! 아마 다음 이슈에서 해당 부분이 사라질 것 같아서 일단 수정은 안해둘게요!